workers/unity/Assets/Playground/Scripts/MonoBehaviours/FlashOnCollision.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Generated.Playground;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using UnityEngine;
+
+public class FlashOnCollision : MonoBehaviour
+{
+    [Require] private Generated.Playground.Collisions.Reader reader;
+
+    private float collideTime;
+    private bool flashing = false;
+    private const float flashTime = 0.2f;
+
+    void OnEnable()
+    {
+        if (reader != null) // Needed until prefab preprocessing is implemented
```

Comment:
Please use a `// TODO:` here with the ticket id for the preprocessing

Reply:
And move the comment to the line above.